### ISSUE TITLE
Additional null-line in Pool

### ISSUE THREAD

**Issue by vJenny** [catboost/catboost#170]
I have data with following shape:
```python
data = pd.read_csv('train.csv', index_col='id')
data.shape
>> (6685, 124)
```
Then I try to create CatBoost's Pool:
```python
pool = Pool('train1.csv', column_description='column_description.cd',
            delimiter=',' ,has_header=True)
print(pool.num_row())
>> 6686
```
As you can see, the shape has changed. And at the end of the pool I have an additional null-object.
```python
features = pool.get_features()
features[-1]
>> [0.0, 0.0, 0.0, ...,  0.0]
```

**Comment by nikitxskv** [comment]
@vJenny, Hello!
We fixed it. Thank you for report!

**Comment by nikitxskv** [comment]
This is fixed in version 0.4.1
